docs/source/docker/advanced/datasource_predictions.rst

```diff
@@ -213,6 +213,19 @@ Predictions for an image must have a `file_name` and `predictions`.
 Here, `file_name` serves as a unique identifier to retrieve the image for which
 the predictions are made and predictions is a list of `Prediction Singletons` for the corresponding task.
 
+- :code:`probabilities` are the per class probabilities of the prediction
+- :code:`score` is the final prediction score/confidence
+
+.. note:: Some frameworks only provide the score as the model output. Having
+          also the class probabilities can be valuable information for 
+          active learning. For example, an object detection model could have a 
+          score of `0.6` and the predicted class is a tree. However, without 
+          class probabilities we cannot know what the prediction margin or 
+          entropy is. With the class 
+          probabilities we would additionally know, whether the model thought 
+          that it's `0.5` tree, `0.4` person and `0.1` car or `0.5` tree, 
+          `0.25` person and `0.25` car.
```

Comment:
What is the relationship between the score and the maximum probability? Are they are equal? Or is their relationship at least monotonously increasing? Or could it happen that a score is very low, but the maximum probability is very high or vice-versa?

Reply:
Depends on how you do the NMS. But usually the score here is object_conf x class conf. Except if you have both separately.

Should I add this info?

Reply:
I think it could help, yes.

Reply:
That explanation is better, thank you :)